roles/network_validated_content/files/project_config_files/tox.ini

```diff
@@ -0,0 +1,34 @@
+[tox]
```

Comment:
Can we replace this with https://github.com/ansible-network/network.telemetry/blob/main/tox.ini?